### ISSUE TITLE
Sessions are still valid after deleting (revoking?) a token

### ISSUE THREAD

**Issue by hanoii** [platformsh/platformsh-cli#491]
I set the cli to use a file token, but then soon remove it so configure a different one, but everything continued to work, even after wiping the platform cli cache.

The only way to make sure the cli doesn't use the old token was to remove .session. It seems the accessToken of the session remains valid.

**Comment by hanoii** [comment]
Removing the session likely only made the client request for a new one, it's very likely the access token that was there is still active until its expiration.

**Comment by pjcdawkins** [comment]
The `platform logout` command is there (it removes `.session`) - but yes, this should be fixed so that the session is invalidated if the configured API token changes.

**Comment by hanoii** [comment]
yes, I didn't even use `platform login`, that's why I was using the token, but yes, it should be.

**Comment by pjcdawkins** [comment]
The PR #492 should improve and clarify this. There are also some changes we can make on the backend to make it easier to manage tokens (that will take longer).
